convert/src/label_studio_to_badgerdoc/models/label_studio_models.py

```diff
@@ -85,18 +85,18 @@ class Prediction(BaseModel):
 
 
 class Annotation(BaseModel):
-    id: int = 7
+    id: int = 1
     completed_by: int = 1
```

Comment:
These values should not be hardcoded with default values according to this:
https://github.com/epam/badgerdoc/pull/117#discussion_r1062214438

Reply:
Fixed